sql/patch.sql

```diff
@@ -47,3 +47,8 @@
 
 --  #EndIf
 --    all blocks are terminated with and #EndIf statement.
+
+#IfRow2D list_options list_id lists option_id apps
+DELETE FROM list_options WHERE list_id="lists" AND "option_id"="apps";
+DELETE FROM list_options WHERE list_id="apps";
+#EndIf
```

Comment:
I'll be removing the `portal/sign/assets/signit.php` script that relied on the app feature in my current portal PR. Just a heads up and no big deal because I deprecated this feature last year.